### PR TITLE
Add Self-contained example for Elasticsearch #788

### PR DESCRIPTION
**Pull request by arunma**
@ennru Updated with the latest from upstream.

### CODE REVIEW

**Comment by ennru** [comment]
Fixes #788

File: doc-examples/src/main/scala/elastic/FetchUsingSlickAndStreamIntoElastic.scala [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import org.apache.http.HttpHost
+import org.elasticsearch.client.RestClient
+import playground.elastic.ElasticsearchMock
+import playground.{ActorSystemAvailable, ElasticSearchEmbedded}
+import spray.json.DefaultJsonProtocol._
+import spray.json.JsonFormat
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.language.postfixOps
```
[the remark this answers is not in the document]
Comment (arunma) [reply doc-examples/src/main/scala/elastic/FetchUsingSlickAndStreamIntoElastic.scala]: :-) you are not alone on this. The postfixOps was annoying but I thought that's how most people prefer it to be. Removing it.

File: doc-examples/src/main/scala/elastic/FetchUsingSlickAndStreamIntoElastic.scala [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        Movie(id, genre, title, gross)
+    }
+
+
+  implicit val client = RestClient.builder(new HttpHost("localhost", 9201)).build()     // (7)
+  implicit val format: JsonFormat[Movie] = jsonFormat4(Movie)                           // (8)
+
+  val elasticSearchSink = ElasticsearchSink.create[Movie]("movie", "boxoffice")         // (9)
+
+  val done: Future[Done] = fetchDataFromTableFlow
```
[the remark this answers is not in the document]
Comment (arunma) [reply doc-examples/src/main/scala/elastic/FetchUsingSlickAndStreamIntoElastic.scala]: Makes perfect sense. For some reason, I thought the segregation looks cleaner. May be it's just for larger flows. Fixing this up as well. Will push an update to this patch shortly. Hopefully, there's no conflicts.

**Comment by arunma** [comment]
@ennru Thanks a lot for your review :-)
Have carried over the code changes and made the paradox reflect the positions.

**Comment by ennru** [comment]
Thank you for your first contribution. Keep going!

**Comment by ScalaWilliam** [comment]
Nice @ennru & @arunma.

**Pull request merged**